codebasin/util.py

```diff
@@ -215,8 +246,9 @@ def _load_toml(file_object: typing.TextIO, schema_name: str) -> object:
 
     Returns
     -------
-    Object
-        The loaded TOML.
+    dict[str, Any]
+        The loaded TOML object, represented as a Python
+        dict with str key/value mappings.
```

Comment:
I haven't thought about this before, so want to double-check you agree.

I think this was set up as `-> object` before because when we were loading JSON, the result was not guaranteed to be a `dict`.  For example, a compilation database is an array of objects.

It does seem like TOML must be key-value pairs, and the documentation says that "TOML is designed to map unambiguously to a hash table".  So I _think_ this is right...?

Reply:
Yup, that's why I left the JSON signature as `object`, but updated it for the TOML. `tomllib.load`'s signature **only** returns `dict[str, Any]`, because I think by design it can't return anything else (i.e. keys **have** to be strings).